packages/@uppy/audio/src/Audio.jsx

```diff
@@ -0,0 +1,368 @@
+import { h } from 'preact'
+
+import { UIPlugin } from '@uppy/core'
+
+import getFileTypeExtension from '@uppy/utils/lib/getFileTypeExtension'
+import supportsMediaRecorder from './supportsMediaRecorder.js'
+import RecordingScreen from './RecordingScreen.jsx'
+import PermissionsScreen from './PermissionsScreen.jsx'
+import locale from './locale.js'
+
+import packageJson from '../package.json'
+
+/**
+ * Audio recording plugin
+ */
+export default class Audio extends UIPlugin {
+  static VERSION = packageJson.version
+
+  #stream = null
+
+  #audioActive = false
+
+  #recordingChunks = null
+
+  #recorder = null
+
+  #capturedMediaFile = null
+
+  #mediaDevices = null
+
+  #supportsUserMedia = null
+
+  constructor (uppy, opts) {
+    super(uppy, opts)
+    this.#mediaDevices = navigator.mediaDevices
+    this.#supportsUserMedia = this.#mediaDevices != null
+    this.id = this.opts.id || 'Audio'
+    this.type = 'acquirer'
+    this.icon = () => (
```

Comment:
Ideally we would have an SVG loader, because this feels cluttered. But I understand that is out of scope for this PR.